reason-react-native/src/apis/Animated.re

```diff
@@ -239,10 +239,26 @@ external loop: Animation.t => Animation.t = "";
 [@bs.module "react-native"] [@bs.scope "Animated"]
 external loopWithConfig: (Animation.t, loopConfig) => Animation.t = "loop";
 
-type animatedEvent;
+type eventOptions;
+[@bs.obj]
+external eventOptions:
+  (~listener: 'event => unit=?, ~useNativeDriver: bool=?, unit) => eventOptions =
+  "";
+
+type eventOptions2;
+[@bs.obj]
+external eventOptions2:
```

Comment:
Do we really need two different eventOptions types? In the worst case, the second listener param would be `undefined` for some event types, right?

Reply:
Yeah it will be undefined as it won't be explicitly defined.

Maybe we can go with an `=?` (not sure if that ok in practice tho)

```reason
type eventOptions;
	[@bs.obj]
	external eventOptions:
	  (~listener: ('event, 'option2=?) => unit=?, ~useNativeDriver: bool=?, unit) => eventOptions =
	  "";
```

Reply:
If the `listener` has to be either `event => unit` or `(event, gestureState) => unit`, why don't we make use of that?

Sorry, I haven't had time to try out `Animated.event` yet, so I can't be more useful.